website/content/docs/commands/secrets/disable.mdx

```diff
@@ -30,3 +30,11 @@ $ vault secrets disable aws/
 
 There are no flags beyond the [standard set of flags](/docs/commands)
 included on all commands.
+
+## Force Disable
+
+Because `secrets disable` revokes secrets associated with this mount, there can
+be errors that prevent the secrets engine from being disabled if part of the
+revocation fails. One can get around this by performing a
+[prefix force revoke](/docs/commands/lease/revoke) on the mount, followed by a
```

Comment:
I would make sure to tell them to first try increasing the timeout.  And put some big warnings around the use of the force flag, since it may lead to dangling credentials (e.g. the dynamic secrets that aren't getting revoked as a result linger, the corresponding db/cloud creds are still around and a potential security issue.)

Reply:
how's this: 7fe721670e

Reply:
I'll throw in my 2 cents as the raiser of the issue referenced and say that it worked. In our case it was a PKI mount so no dangly bits.